packages/teleport/src/services/ssh/types.ts

```diff
@@ -16,10 +16,10 @@ limitations under the License.
 
 export interface Participant {
   user: string;
-  remoteAddr: string;
 }
 
 export interface Session {
+  kind: SessionKind;
```

Comment:
It's imprecise to export this `Session` from `services/ssh` and then also use it for k8s sessions. What about adding something like this to `services/kube/types.ts`

```typescript
import { Session } from 'teleport/services/ssh';

export interface KubeSession extends Session {
  kubeClusterId: string;
}
```

Reply:
i meant to rename this file to `session`, and rename the existing `session` to `websession`, see here: https://github.com/gravitational/webapps/pull/972/commits/de7bddd306bd74b05d60a8ba978c0d937910acd3

Reply:
sigh... renaming things kinda screwed up the file diff... just fyi